backend/src/controllers/questionAttempt.js

```diff
@@ -77,12 +77,61 @@ const getQuestionAttempts = async (req, res) => {
       },
     ],
     order: [['updatedAt', 'DESC']],
-    subQuery: false,
+    //limit: pageSize || 50,
   });
 
   res.status(200).send({ data: questionAttempts });
 };
 
-module.exports = {
-  getQuestionAttempts,
+const getQuestionAttemptsID = async (req, res) => {
```

Comment:
deze endpoint wordt niet aangeroepen, dus denk niet dat ie nog nodig is?

Reply:
oh shit ja ik was vergeten op te schonenn met niet gebruikte dingen, ga ik gelijk ff doen